site/en/contribute/breaking-changes.md

```diff
@@ -80,41 +78,41 @@ as the code change.
 
 ## Labels {:#labels}
 
-Once the commit is merged, add the label
+Once the commit is merged and the incompatible change is ready to be adopted, add the label
 [`migration-ready`](https://github.com/bazelbuild/bazel/labels/migration-ready){: .external}
 to the GitHub issue.
 
-Later a [Bazel release manager](https://github.com/bazelbuild/continuous-integration/blob/master/docs/release-playbook.md){: .external}
-will update the issue and replace the label with `migration-xx.yy`.
-
-The label `breaking-change-xx.yy` communicates when we plan to flip the flag. If
-a migration window needs to be extended, the author updates the label on GitHub
-issue accordingly.
-
 If a problem is found with the flag and users are not expected to migrate yet:
-remove the flags `migration-xx.yy`.
+remove the flags `migration-ready`.
 
 ## Updating repositories {:#update-repos}
 
-1. Ensure that the core repositories are migrated. On the
-  [`bazelisk-plus-incompatible-flags` pipeline](https://buildkite.com/bazel/bazelisk-plus-incompatible-flags){: .external},
-  the flag should appear under "The following flags didn't break any passing Bazel team owned/co-owned projects".
-1. Notify the owners of the other repositories.
-1. Wait 14 days after the notifications to proceed, or until the flag is under
-   "The following flags didn't break any passing projects" in the CI.
+Bazel CI tests a list of important projects at [Bazel@HEAD + Downstream](https://buildkite.com/bazel/bazel-at-head-plus-downstream). Most of them are often dependencies of other Bazel projects, therefore it's important to migrate them to unblock the migration for the broader community.
+
+To monitor the migration status of those projects, you can use the [`bazelisk-plus-incompatible-flags` pipeline](https://buildkite.com/bazel/bazelisk-plus-incompatible-flags), check how this pipeline works [here](https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#checking-incompatible-changes-status-for-downstream-projects).
+
+Migrating projects in the downstream pipeline is NOT entirely the responsibility of the incompatible change author. But you can do the following to accelerate the migration and make life easier for both Bazel users and the Bazel Green Team.
+
+1. File Github issues to notify the owners of the downstream projects broken by your incompatible change.
+1. Send PRs to fix downstream projects.
+1. Reach out to the Bazel community for help on migration (e.g. [Bazel Rules Authors SIG](https://bazel-contrib.github.io/SIG-rules-authors/)).
 
 ## Flipping the flag {:#flip-flag}
 
 Before flipping the default value of the flag to true, please make sure that:
 
-  * The migration window is respected.
+  * Core repositories in the ecosystem are migrated. On the
+  [`bazelisk-plus-incompatible-flags` pipeline](https://buildkite.com/bazel/bazelisk-plus-incompatible-flags){: .external},
```

Comment:
you might need to indent this line and the next

Reply:
Thanks, done!